api/model/anime.go

```diff
@@ -0,0 +1,8 @@
+package model
+
+type AnimeStruct struct {
```

Comment:
It is redundant to name it AnimeStruct.
```suggestion
type Anime struct {
```